e2e/cypress/tests/integration/keyboard_shortcuts/ctrl_cmd_I_toggles_channel_info_rhs_spec.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @keyboard_shortcuts
+
+describe('Keyboard Shortcuts', () => {
+    let url;
+
+    before(() => {
+        cy.apiInitSetup({loginAfter: true}).then(({channelUrl}) => {
+            url = channelUrl;
+        });
+    });
+
+    it('CTRL/CMD+SHIFT+I - Toggle Channel Info RHS', () => {
```

Comment:
```suggestion
    it('MM-T4775 CTRL/CMD+SHIFT+I - Toggle Channel Info RHS', () => {
```

Reply:
I've added the test case for this at https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T4775

Reply:
Cool thanks!